contrib/android/config/android.h

```diff
@@ -26,6 +26,18 @@
 #ifndef ARCHIVE_PLATFORM_H_ANDROID_INCLUDED
 #define	ARCHIVE_PLATFORM_H_ANDROID_INCLUDED
 
+#include <android/api-level.h>
+#ifdef __ANDROID_API__
```

Comment:
Yes that's true http://stackoverflow.com/a/5085425

Thanks for pointing that out. But since you already merged it I guess it's not too important.